src/components/UserDataUploads.tsx

```diff
@@ -36,36 +38,99 @@ const UserDataUploadComponent = ({
-                                    accept=".csv,.xlsx,.xls"
-                                    aria-describedby="csv-upload-hint"
-                                />
-                            </FormElementWrapper>
-                        </div>
+                        {showPriceOption && (
+                            <FormGroupWrapper errors={errors} errorId="pounds">
+                                <fieldset className="govuk-fieldset">
+                                    <legend className="govuk-fieldset__legend govuk-fieldset__legend--m">
+                                        <h2 className="govuk-fieldset__heading" id="passenger-type-page-heading">
+                                            Are prices in pounds or pence?
+                                        </h2>
+                                    </legend>
+                                    <FormElementWrapper
+                                        errors={errors}
+                                        errorId="pounds"
+                                        errorClass="govuk-radios--error"
+                                    >
+                                        <div className="govuk-radios">
+                                            <div className="govuk-radios__item">
+                                                <input
+                                                    className="govuk-radios__input"
+                                                    id="pounds"
+                                                    name="poundsOrPence"
+                                                    type="radio"
+                                                    value="pounds"
+                                                    defaultChecked={poundsOrPence === 'pounds'}
+                                                />
+
+                                                <label className="govuk-label govuk-radios__label" htmlFor="pounds">
+                                                    Pounds, for example 2.50
+                                                </label>
+                                            </div>
+                                            <div className="govuk-radios__item">
+                                                <input
+                                                    className="govuk-radios__input"
+                                                    id="pence"
+                                                    name="poundsOrPence"
+                                                    type="radio"
+                                                    value="pence"
+                                                    defaultChecked={poundsOrPence === 'pence'}
+                                                />
+                                                <label className="govuk-label govuk-radios__label" htmlFor="pence">
+                                                    Pence, for example 250
```

Comment:
Looking around the site, if we give examples or guidance within the options, we surround it with brackets, e.g. Pence (for example 250) etc.